common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -56,16 +56,8 @@
     </fieldset>
     <div class="indicator-container">
         % if input_type == 'checkbox' or not value:
-            <span class="status ${status.classname if show_correctness != 'never' else 'unanswered'}" id="status_${id}" aria-describedby="inputtype_${id}" data-tooltip="${status.display_tooltip}">
-                <span class="sr">
-                    %for choice_id, choice_label in choices:
-                        % if choice_id in value:
-                            ${choice_label},
-                        %endif
-                    %endfor
-                    -
-                    ${status.display_tooltip}
-                </span>
```

Comment:
@muhammad-ammar see @lamagnifica's comment here:
https://github.com/edx/edx-platform/pull/12691#issuecomment-242732054

I think the tooltip should match the SR text (as @lamagnifica suggests): "Not yet answered"